src/index.ts

```diff
@@ -373,6 +373,15 @@ You may find the requirements here: ${cyan('https://tauri.app/v1/guides/getting-
   }
 };
 
+function toValidPackageName(appName: string) {
+  return appName
+    .trim()
+    .toLowerCase()
+    .replace(/\s+/g, '-')
+    .replace(/^[._]/, '')
+    .replace(/[^a-z0-9-~]+/g, '-')
```

Comment:
If I understand this regex correctly, this replaces any character that isn't alphanumeric, `-`, `~` with `-`?
I think:
- `~` should be also converted to `-`
- capital letters should be fine to include too

Reply:
The package name is allowed to include `~`, but no capital letters.

Reply:
`~` is not acceptable in the crate name so it should be removed too

Reply:
I tried it and `~` seems to be allowed 🤔

Reply:
it is not allowed as a crate name in `src-tauri/Caro.toml` but allowed in `package.json`, but since we use `app_name` for both, we gotta use the more restriected rules so we need to also remove `~`.

Reply:
ok, thank you for the explanation.